README.md

```diff
@@ -147,3 +147,14 @@ Clusterman is licensed under the Apache License, Version 2.0: http://www.apache.
 Everyone is encouraged to contribute to Clusterman by forking the
 [Github repository](http://github.com/Yelp/clusterman) and making a pull request or
 opening an issue.  Please read our [Code of Conduct](https://github.com/Yelp/clusterman/code-of-conduct.md).
+
+### Instructions for Yelp developers
+
+0) Run `git add remote yelp git@git.yelpcorp.com:services/clusterman` (you only need to do this once)
+1) Make your changes and push a branch to GitHub for PR
+2) Once your PR is approved, merge your changes to master
+3) In your local repository run
+    git pull origin master
+    git push yelp master
+
+This will trigger the Jenkins pipleine and you can run the rest of the deploy as normal.
```

Comment:
How about: "This will trigger the Jenkins pipeline, which will deploy Clusterman as normal".

(also pipeline is mispelled as "pipleine")